anydex/core/pricelevel_list.py

```diff
@@ -17,8 +19,7 @@ def insert(self, price_level: PriceLevel) -> None:  # type: (PriceLevel) -> None
         """
         :type price_level: PriceLevel
         """
-        self._price_list.append(price_level.price)
-        self._price_list.sort()
+        bisect.insort(self._price_list, price_level.price)
```

Comment:
Very nice change! I remember that I already did some optimizations on the order book but I apparently missed this important one 😀

Reply:
Thank you! Happy to help :smiley: